<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1083</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="main_frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>768</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#main_frame{
background-color: rgb(255,255,255);
}

</string>
   </property>
   <widget class="QFrame" name="Wordcloud_box">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>371</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: '#D2E1FF';
boarder-color: rgb(0,0,0);
border-radius: 15px;
</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="Wordcloud">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="return_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbox">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>50</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="graph_box1">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>100</y>
      <width>311</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: '#D2E1FF';
boarder-color: rgb(0,0,0);
border-radius: 15px;
</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGraphicsView" name="graph_1"/>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="graphbox_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>100</y>
      <width>341</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: '#D2E1FF';
boarder-color: rgb(0,0,0);
border-radius: 15px;
</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGraphicsView" name="graph_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="logo_box">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>330</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: '#D2E1FF';
boarder-color: rgb(0,0,0);
border-radius: 15px;
</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="olympic_logo">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="graph_box4">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>330</y>
      <width>441</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: '#D2E1FF';
boarder-color: rgb(0,0,0);
border-radius: 15px;
</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QGraphicsView" name="graph_4"/>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="graph_box3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>591</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: '#D2E1FF';
boarder-color: rgb(0,0,0);
border-radius: 15px;
</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QGraphicsView" name="graph_3"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="search_btn">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>20</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>검색</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>20</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;종목을 선택해주세요&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
